src/python/pants/notes/2.15.x.md

```diff
@@ -1,5 +1,37 @@
 # 2.15.x Release Series
 
+## 2.15.0.dev4 (Oct 07, 2022)
+
+### New Features
+
+* Optionally export venvs via symlink into the pex_root. ([#17130](https://github.com/pantsbuild/pants/pull/17130))
+
+* go: add cgo support for tailor ([#17087](https://github.com/pantsbuild/pants/pull/17087))
+
+### User API Changes
+
+* Upgrade Pex to 2.1.108. ([#17112](https://github.com/pantsbuild/pants/pull/17112))
+
+### Plugin API Changes
+
+* Switch `LintRequest.name` to `.tool_subsystem` ([#17103](https://github.com/pantsbuild/pants/pull/17103))
+
+* Rename/Refactor `style_request` to better capture intent. ([#17084](https://github.com/pantsbuild/pants/pull/17084))
+
+### Bug fixes
+
+* [internal] Move pyoxidized pants binary out of the main source tree ([#16703](https://github.com/pantsbuild/pants/pull/16703))
```

Comment:
@chrisjrn maybe the [internal] vs bugfix label was a deliberate communication of nuance. I was not sure so left as is.

Reply:
@jsirois _internal_ was deliberate, as the pyox option is very very experimental

Reply:
Ok, I should have ignored the label then :/ Sorry about that - hard to guess.

Reply:
This probably should have been in the internal category, not bug fix -- afaict this change didn't impact end users. The changelog is solely intended to be notes for end users, not Pants devs

I'll adjust during my release duty today